filters/filters.txt

```diff
@@ -23831,6 +23831,7 @@ trendsderzukunft.de##+js(aopr, onload)
 ! https://github.com/NanoMeow/QuickReports/issues/3117
 @@||vidlii.com^$ghide
 vidlii.com##+js(nostif, cloudflare-app)
+vidlii.com##.adsbygoogle:style(display:none !important;)
```

Comment:
Why the :style filter?
`vidlii.com##.adsbygoogle`
should work just fine.

Reply:
I forgot about that.